### DISCUSSION TITLE
请教nginx代理https应该如何做？

### DISCUSSION THREAD

**Discussion by leqii-com** [mediacms-io/mediacms#1119] [Q&A]
ENV: 环境是Ubuntu24.04 . Docker ce 26.1.4

我配置了https代理，提示如下类似错误：
_commons.js:2 Mixed Content: The page at 'https://qrcms.test.cn/' was loaded over HTTPS, but requested an insecure XMLHttpRequest endpoint 'http://qrcms.test.cn/api/v1/media?show=featured'. This request has been blocked; the content must be served over HTTPS.

我是需要修改后端接口代码？还是修改某个配置？是否可以通过修改nginx代理配置修复这个问题？

**Comment by mgogoulos** [comment]
Hey what's that? I don't speak Chinese
